docs/src/user-guide/quick-start/clp-text.md

```
@@ -0,0 +1,157 @@
+java.lang.IllegalArgumentException: java.net.UnknownHostException: i-e5d112ea
+    at org.apache.hadoop.security.buildTokenService(SecurityUtil.java:374)
+    at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2033)
+Caused by: java.net.UnknownHostException: i-e5d112ea
+    ... 17 more
+```
+
++++
+**Figure 2**: A set of unstructured text log events.
+:::
+
+The query in [Figure 2](#figure-2) will match with the first log message, as the `?` will match the
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)